apps/server/src/shared/domain/entity/system.entity.ts

```diff
@@ -61,6 +62,9 @@ export class OauthConfig {
 
 	@Property()
 	jwksEndpoint: string;
+
+	@Property()
+	provisioningUrl: string;
```

Comment:
is this a mandatory field?

Reply:
it's optional now, because it should be ;-)